client/js/store/statement/Fragment.js

```diff
@@ -482,17 +483,17 @@ export default {
           // If the reviewer has been set, update fragment assignment
           if (hasOwnProp(data, 'departmentId')) {
             if (hasOwnProp(data, 'lastClaimed') && hasOwnProp(responseRelationships, 'lastClaimedUser')) {
-              dataToUpdate.lastClaimedUserId = responseRelationships.lastClaimedUser.data.id
+              dataToUpdate.lastClaimedUserId = responserelationships.LastClaimedUser.data.id
             }
 
-            dataToUpdate.departmentId = hasOwnProp(responseRelationships, 'department') ? responseRelationships.department.data.id : ''
+            dataToUpdate.departmentId = hasOwnProp(responseRelationships, 'department') ? responserelationships.Department.data.id : ''
 
             if (dataToUpdate.departmentId) { // If departmentId is in response and is not null
               // we reset the assignee with the values from BE
-              if (responseRelationships.assignee?.data) {
-                const newAssigneeId = responseRelationships.assignee.data.id
+              if (responserelationships.Assignee?.data) {
```

Comment:
So when do we have to use pascal case for relationships, and when camel case?

Reply:
Looks like i missed to re-transform the relationships within js-files.
relationships have to be camels